src/sage_docbuild/conf.py

```diff
@@ -374,7 +374,7 @@ def set_intersphinx_mappings(app, config):
 if os.environ.get('SAGE_USE_CDNS', 'no') == 'yes':
     mathjax_path = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"
 else:
-    mathjax_path = 'mathjax/tex-chtml.js'
+    mathjax_path = 'tex-chtml.js'
     html_common_static_path += [MATHJAX_DIR]
```

Comment:
This change does not break the documentation, but has a problem which was the reason of the prefix `mathjax/`.

This is the content of the directory `_static` built by Sphinx:
```bash
$ cd _static/
$ ls
_sphinx_javascript_frameworks_compat.js mml-svg.js
a11y                                    node-main.js
adaptors                                output
basic.css                               pdf.png
check-solid.svg                         plot_directive.css
clipboard.min.js                        plus.png
copy-button.svg                         pygments.css
copybutton.css                          sageicon.png
copybutton.js                           sagelogo.png
copybutton_funcs.js                     scripts
core.js                                 searchtools.js
custom-furo.css                         skeleton.css
debug.css                               sphinx_highlight.js
doctools.js                             sre
documentation_options.js                startup.js
favicon.ico                             styles
file.png                                tex-chtml-full.js
input                                   tex-chtml.js
jquery-3.6.0.js                         tex-mml-chtml.js
jquery.js                               tex-mml-svg.js
jupyter-sphinx.css                      tex-svg-full.js
language_data.js                        tex-svg.js
latest.js                               thebe-sage.js
loader.js                               thebelab-helper.js
logo_sagemath_black.svg                 thebelab.css
logo_sagemath_white.svg                 ui
minus.png                               underscore-1.13.1.js
mml-chtml.js                            underscore.js
```
Note that all mathjax files are mixed with other static files while the original `_static` directory looks like this:
```bash
$ cd _static/
$ ls
_sphinx_javascript_frameworks_compat.js logo_sagemath_white.svg
basic.css                               mathjax  <----------------------------------
check-solid.svg                         minus.png
clipboard.min.js                        pdf.png
copy-button.svg                         plot_directive.css
copybutton.css                          plus.png
copybutton.js                           pygments.css
copybutton_funcs.js                     sageicon.png
custom-furo.css                         sagelogo.png
debug.css                               scripts
doctools.js                             searchtools.js
documentation_options.js                skeleton.css
favicon.ico                             sphinx_highlight.js
file.png                                styles
jquery-3.6.0.js                         thebe-sage.js
jquery.js                               thebelab-helper.js
jupyter-sphinx.css                      thebelab.css
language_data.js                        underscore-1.13.1.js
logo_sagemath_black.svg                 underscore.js
```
Note that all mathjax files in the subdirectory `mathjax`. 

I think this is a regression.

Reply:
> This change does not break the documentation, but has a problem which is the reason of the prefix `mathjax/`...
>
> Note that all mathjax files are mixed with other static files while the original (without this PR) `_static` directory looks like this:
> 
>...
>
> Note that all mathjax files are in the subdirectory `mathjax`.
> 
> I think this is a regression.

It's messy, but does it hurt anything?

That directory contains all of sphinx's build files and it's sphinx that decides what to put in there. We also see jquery.js and underscore.js (two independent projects) in that directory. If it were up to me the images, css, and javascript would all be in a separate directory, but I don't think moving just one javascript file into a subfolder makes things any cleaner or more consistent.

In any case, I changed it because the system mathjax may not live in a directory called "mathjax". Do you see an easy way around that?

Reply:
Yes, `_static` directory is already a mess, but now mathjax becomes a mess too. mathjax comprises of many javascript files and directories containing data files. Mixing all of them with other files in `_static` directory makes the situation worse. We do (I did) look into `_static` directory from time to time to fix things. It's better to keep it tidy as much as we can.